package.json

```diff
@@ -32,7 +32,7 @@
     "format:check": "prettier --check \"**/*.+(json|yml|yaml|css|md)\"",
     "flow:check": "flow check",
     "concurrently": "concurrently",
-    "postinstall": "node -e \"process.env.NODE_ENV != 'production' && process.exit(1)\" || husky install"
+    "prebublish": "husky install"
```

Comment:
There's a small typo "prepublish"

Reply:
good catch, wonder why it worked ...

Reply:
@saulipurhonen i will revert to `postinstall` `prepublish`  gives:

```
➜ npm install --production                             

> metadata-submitter-frontend@0.8.0 prepublish
> husky install

sh: husky: command not found
npm ERR! code 127
npm ERR! path /home/path/metadata-submitter-frontend
npm ERR! command failed
npm ERR! command sh -c husky install
```

Reply:
Alright.
Just checked **Husky**  documentation and it's installed via `prepare`, not `prepublish`

Reply:
ok, so is it ok to leave like that, `prepare` gives the same or any other ideas ?

Reply:
`prepare` seems to work locally via `npm install --production` but this I think this isn't that important issue. Only perk I see is is that it looks lot of cleaner.

Reply:
From the npm docs I just read, `prepublish` seems to be deprecated and `prepare` is another option.

Reply:
figured out another solution with `pinst` https://github.com/typicode/husky/issues/867#issuecomment-782845423 - lets see what CI has to say

Reply:
... and it seems to like it. I will try it in docker the same, and it keeps it cleaner, and I will think where to document this in a future PR.